js/index.js

```diff
@@ -0,0 +1,191 @@
+  },
+
+  {
+    id: '3',
+    artistImage: './resources/vir-das.jpg',
+    artistName: 'Vir Das',
+    imageAlt: 'Picture of Vir Das',
+    artistLabel: 'Stand-up comedian, actor, and musician.',
+    latestRelease:
+      'Vir Das just released a new comedy special titled "Losing It" on Netflix.',
+  },
+
+  {
+    id: '4',
+    artistImage: './resources/anubhav-bassi.jpg',
+    artistName: 'Anubhav Bassi',
+    imageAlt: 'Picture of Anubhav Bassi',
+    artistLabel: 'Stand-up comedian and content creator.',
+    latestRelease:
+      'Anubhav Bassi\'s latest comedy video "Rishte Wale" is now trending on YouTube.',
+  },
+
+  {
+    id: '5',
+    artistImage: './resources/rohan-joshi.jpg',
+    artistName: 'Rohan Joshi',
+    imageAlt: 'Picture of Rohan Joshi',
+    artistLabel: 'Stand-up comedian and writer.',
+    latestRelease:
+      'Rohan Joshi\'s new stand-up special "Wake N Bake" is receiving rave reviews.',
+  },
+  {
+    id: '6',
+    artistImage: './resources/harsh-gujral.jpg',
+    artistName: 'Harsh Gujral',
+    imageAlt: 'Picture of Harsh Gujral',
+    artistLabel: 'Stand-up comedian and entertainer.',
+    latestRelease:
+      'Harsh Gujral just released a hilarious new comedy video titled "Jokes Unlimited".',
+  },
+];
+
+// Create the constant header of the page
+const footer = document.createElement('footer');
+footer.className = 'whole-footer-display';
+footer.innerHTML = `
+<section class='footer'>
+  <h3 class="footer-head font">Featured Artists</h3>
+  <div class="red-line-2 font">_______</div>
+</section>`;
+body.appendChild(footer);
+
+// Create a function and create the element for mobile display
+function showFeaturedArtistsMobile(info) {
+  const shownArtistMobile = document.createElement('section');
+  shownArtistMobile.className = 'featured-artists-container';
+  shownArtistMobile.innerHTML = `<section class="featured-artists-container for-phone">
+  <div class="featured-artists-item-div for-phone">
+      <img class="background-top for-phone" src="resources/icons/Picture-top.png" alt="Picture background" role="presentation">
+      <img class="featured-artists-image for-phone" src='${featuredArtists[info].artistImage}' alt="Picture of Haywaya">
+      <div class="featured-artists-body-div for-phone">
+          <h4 class="font for-phone">${featuredArtists[info].artistName}</h4>
+          <h5 class="font for-phone"> <i>${featuredArtists[info].artistLabel}</i></h5>
+          <div class=" for-phone">__</div>
+          <h6 class="font for-phone">
+              ${featuredArtists[info].latestRelease}
+          </h6>
+      </div>
+  </div>
+</section>`;
+  footer.appendChild(shownArtistMobile);
+}
+
+const shownArtistDeskContainer = document.createElement('section');
+shownArtistDeskContainer.className = 'desk-featured-artists-container for-desktop';
+// shownArtistDeskContainer.innerHTML =
+// `<div class="desk-featured-artists-item-div for-desktop">
+// </div>`,
+footer.appendChild(shownArtistDeskContainer);
```

Comment:
- [ ] Kindly delete any commented code as it conflicts with the best practices and having a clean code.

Reply:
Done 👍 @Ofarouq310